app/config/frontendAppConfig.scala

```diff
@@ -112,6 +113,9 @@ class FrontendAppConfig @Inject()(val runModeConfiguration: Configuration, val e
   private lazy val paymentsReturnBase: String = getString(Keys.paymentsReturnBase)
   override lazy val paymentsReturnUrl: String = paymentsReturnBase + getString(Keys.paymentsReturnUrl)
 
+  private lazy val unauthenticatedPaymentsBase: String = getString(Keys.unauthenticatedPaymentsBase)
```

Comment:
Unnecessary creation of base value as it's not used multiple times.

Rewriting
```
override lazy val unauthenticatedPaymentsUrl: String = unauthenticatedPaymentsBase + getString(Keys.unauthenticatedPaymentsUrl)
```
to
```
override lazy val unauthenticatedPaymentsUrl: String = getString(Keys.unauthenticatedPaymentsBase) + getString(Keys.unauthenticatedPaymentsUrl)
```

is not that difficult to read